website/docs/packages/reference/query-module.md

```diff
@@ -63,63 +79,80 @@ When enabled, this property allows the query to automatically execute each time
 
 <dd>
 
-When enabled, this property displays a confirmation modal each time before refreshing the data. This ensures that users have the opportunity to confirm their action before executing the query, preventing unintended data refreshes.
+When enabled, this property displays a confirmation modal each time a query is executed. This ensures that users can confirm their action before executing the query, preventing unintended data updates.
 
-</dd>
 
 
-## JS module
 
-These properties are customizable options available within the JS editor of the JS module. They can be accessed by adding the JS module in the App.
 
-Additionally, you can add multiple JS modules from the same package, each with different configurations.
 
+</dd>
 
-<ZoomImage
-  src="/img/js-module-setting.png" 
-  alt="JS modules image"
-  caption=""
-/>
 
-### Function setting
+## JS module instance
 
-This setting allows you to configure each function available in the JS module according to your requirements.
+These properties are customizable options available within the JS editor of the JS module. They can be accessed by adding the JS module to your app. When you add multiple JS modules, whether from the same package or different ones, you can configure each module instance using these properties to meet your specific needs.
 
-#### Function name
 
-<dd>
-
-This setting displays all the function names available in the JS module. 
 
+### Parameters 
 
+This property displays all the parameters available for the specified function.
 
-</dd>
 
-#### Run on page load
+#### Inputs
 
 <dd>
 
-When enabled, this property allows the specified JS function to run when the page loads. You can set this property for each JS function available.
+The input property allows you to pass dynamic values from your app to the JS module. The input parameters are only available if they are defined in the JS function. You cannot edit the input name or JS module configuration from the App; you can only pass values to the available inputs. 
+
+You can pass data either using this **input** property or by providing values during runtime inside an event `(e.g., authUtils.login('email@domain.com', 'dsoi3$dfssn');)`. If both methods are used, the runtime parameters will take precedence over the values set in the UI.
+
+*Example*: If you have a JS function with predefined parameters like `email` and `passwordHash`, and you want to pass Input widget data from the app to the JS module, you can use the **Inputs** property.
+
+```js
+//JS Module
+export default {
+  myFunction: (email, passwordHash) => {
+    console.log("Parameter 1: ", email);
+    console.log("Parameter 2: ", passwordHash);
+    // Perform operations using param1 and param2
+  }
+}
+```
+
+```js
+//JS module instance
+email: 
+{{email_input.text}}
 
+passwordHash:
+{{pass_input.text}}
```

Comment:
Suggestion: what if the binding has a JSObject function that returns the hash based on the argument that is the pass_input.text? In that way, it seems more real the example and also demonstrates that people are also able to access JSObjects from those inputs.